<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главная</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Столик</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Выполнен</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Оплачен</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="add_order_pushButton">
        <property name="text">
         <string>Добавить заказ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open_order_status_button">
        <property name="text">
         <string>Изменить статус</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_order_button">
        <property name="text">
         <string>Управление заказами</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logout_pushButton">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>